mungegithub/mungers/block_paths.go

```diff
@@ -66,6 +67,10 @@ func (b *BlockPath) RequiredFeatures() []string { return []string{} }
 
 // Initialize will initialize the munger
 func (b *BlockPath) Initialize(config *github.Config, features *features.Features) error {
+	return b.loadConfig()
+}
+
+func (b *BlockPath) loadConfig() error {
```

Comment:
this has fatal errors inside it-- I guess that means changing the configmap could trigger crashlooping?